internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -865,12 +870,49 @@ func convertQueue(q manifest.SQSQueue) *template.SQSQueue {
 	if q.IsEmpty() {
 		return nil
 	}
-	return &template.SQSQueue{
+
+	queue := &template.SQSQueue{
 		Retention:  convertRetention(q.Retention),
 		Delay:      convertDelay(q.Delay),
 		Timeout:    convertTimeout(q.Timeout),
 		DeadLetter: convertDeadLetter(q.DeadLetter),
 	}
+
+	if q.FIFO.IsEmpty() {
+		return queue
+	}
+
+	if aws.BoolValue(q.FIFO.Enable) {
+		queue.FIFOQueueConfig = &template.FIFOQueueConfig{
+			Enable: q.FIFO.Enable,
+		}
+		return queue
```

Comment:
Can we do like SNS and not have `Enable` as field in the `template` type:
```suggestion
	queue.FIFOQueueConfig = &template.FIFOQueueConfig{}
	if q.FIFO.IsEnabled() && q.FIFO.Advanced.IsEmpty() {
		return queue
```

Reply:
Yes I was working on the same thing here when I got that comment in SNS PR. The change is coming in the latest commit.